src/themes/minimal/palette.ts

```diff
@@ -180,12 +180,26 @@ const palette = {
 		text: { primary: GREY[800], secondary: GREY[600], disabled: GREY[500] },
 		background: { paper: '#fff', default: '#fff', neutral: GREY[200] },
 		action: { active: GREY[600], ...COMMON.action },
+		proposalStates: {
```

Comment:
#195 @mikeplanzer can you check please these colors for each status of a proposal?